assets/css/admin.scss

```diff
@@ -727,6 +749,125 @@ tr.email-setting-row {
 		padding: 5px;
 	}
 }
+/**
+ * Extensions page
+ */
+.wp_job_manager_add_ons_wrap {
+	.wpjm-extensions-filter-search {
+		display: flex;
+		margin-top: 18px;
+		.wpjm-extension-search-input {
+			padding: 9px 12px;
+			border-radius: 2px;
+			line-height: 20px;
+			font-size: 13px;
+		}
+		.wpjm-extension-search-button {
+			padding: 5px 12px;
+			background: #ffffff;
+		}
+		.subsubsub {
+			text-align: left;
+			margin: 0;
+			align-self: end;
+			li {
+				margin: 0 0 6px 0;
+				a {
+					padding: 6px 8px;
+					background: #ffffff;
+					border-radius: 2px;
+					color: #3C434A;
+					font-weight: 400;
+					margin-right: 4px;
+					font-size: 13px;
+					&.current {
+						background: #1E1E1E;
+						color: #ffffff;
+					}
+				}
+			}
+		}
+		.extension-search {
+			margin-left: auto;
+			align-self: center;
+			display: flex;
+		}
+	}
+}
+@media screen and (max-width: 782px) {
+	.wp_job_manager_add_ons_wrap .wpjm-extensions-filter-search {
+		flex-direction: column-reverse;
+		gap: 18px;
+
+		.subsubsub {
+			align-self: start;
+		}
+		.extension-search {
+			margin-right: auto;
+			margin-left: 0;
+		}
+		.wpjm-extension-search-button {
+			padding: 0 15px;
+			margin-bottom: 0;
+		}
+	}
+}
+@media screen and (max-width: 428px ) {
+	.wp_job_manager_add_ons_wrap .wpjm-extensions-filter-search {
+		.extension-search {
+			// display: flex;
```

Comment:
Commented out code

Reply:
Fixed in 4dfe1c4a420375f6331ef67ed7fd468dfa2c1170